### DISCUSSION TITLE
Let's disable discussion

### DISCUSSION THREAD

**Discussion by mweststrate** [mobxjs/mobx#3469] [General]
Hey folks, I think discussions are heaving a negative effect on support load and I propose to just stick to the issue tracker instead? Discussions are now primarily used as lightweight issues ("X doesn't work (as I expected)") to escape the strict issue format, and often we just need to ask for reproduction etc first, which the issue tracker all nicely handles for us. Als discussions don't really close, seemingly sticking around forever. Since with the current stability there are little conceptual discussions, I think we can just keep using forever open issues for these like we did in the past?

cc
@kubk @urugator

**Comment by kubk** [comment]
Yes, we can disable discussions to force people provide reproductions. I hope the action won't remove old discussions?

**Comment by kubk** [comment]
We need old discussions to convert some of them to issues back (like this one https://github.com/mobxjs/mobx/discussions/681) and to leave some useful information for people (example https://github.com/mobxjs/mobx/discussions/3216)

But maybe if the information is useful, it worth moving to documentation

**Comment by kubk** [comment]
I've done a quick testing with my own repo. Once you disable the Discussion section, all the discussions become invisible, all the URLs give 404 status. You can still resurrect them by enabling the Discussion section, they won't disappear. So we can create a migration plan (convert some discussions into issues or documentation) and then we can close it

**Comment by urugator** [comment]
I think you're approaching discussions in the wrong way :) Or maybe I am, you tell me :D
Firstly I think it should be easy to simply ask a question - one shouldn't have to fill a form. 
At the same time we shouldn't feel obliged nor it should be expected from us to answer or to manage every discussion.
Whether the user gets an answer largely depends on how well he asks, it's his fight. It's ok to ignore discussion, we don't have to ask for reproduction or anything. It also doesn't have to be us replying. Discussions aren't adressed to maintainers. I treat discussions as "free" communication channel for community. It's better than eg chat as it's easier to search, navigate, manage and it's part of the repo.
Low entry barrier, higher uncertainty of recieving a response and wider audience is in my eyes what differentiates discussions from issues and also what makes them useful.

The only thing that annoys me (to stay polite), is when someone seeks attention by immediately creating both exactly the same discussion and an issue.

Lastly category templates for discussions are on the roadmap:
https://github.com/github-community/community/discussions/2838#discussioncomment-261382

**Reply by mweststrate** [reply]
> Firstly I think it should be easy to simply ask a question - one shouldn't have to fill a form.

I agree, but 90% of even the most mundane questions don't provide enough context by default it seems, which, with issue templates is much less of an issue?

> nor it should be expected from us to answer or to manage every discussion.

But I think people do, especially if something isn't working. Could we fine tune discussion templates, to make clear that "something doesn't work" / "I don't understand this behavior" isn't the right question for a discussion item?

> Discussions aren't adressed to maintainers. I treat discussions as "free" communication channel for community

I think this is a great mental model, and probably how it is intended, but I doubt it is how most users look at it? Would be great if we could make that more clear. For now it feels mostly as a Stack Overflow competitor, and maybe the nice think about SO is that it is at least clear it is not maintained by the MobX maintainers :) https://stackoverflow.com/questions/tagged/mobx (that is more than I expected btw, haven't been there in years)
